cluster/ci/config/prow/openshift/jenkins_operator.yaml

```diff
@@ -164,3 +167,81 @@ objects:
           kind: ImageStreamTag
           name: "${NAME}:latest"
       type: ImageChange
+
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: "${INTERNAL_NAME}"
+    annotations:
+      prometheus.io/scrape: "true"
+  spec:
+    selector:
+      app: prow
+      component: "${INTERNAL_NAME}"
+    ports:
+    - port: 80
+      targetPort: 8080
+    type: ClusterIP
+- apiVersion: apps.openshift.io/v1
+  kind: DeploymentConfig
+  metadata:
+    name: "${INTERNAL_NAME}"
+    labels:
+      app: prow
+      component: "${INTERNAL_NAME}"
+  spec:
+    # Do not scale up.
+    replicas: 1
+    revisionHistoryLimit: 2
+    strategy:
+      type: Recreate
+    template:
+      metadata:
+        labels:
+          app: prow
+          component: "${INTERNAL_NAME}"
+      spec:
+        serviceAccountName: "${NAME}"
+        containers:
+        - name: "${NAME}"
+          image: "${NAME}"
+          args:
+          - --jenkins-url=https://ci.dev.openshift.redhat.com/jenkins
+          - --jenkins-user=openshift-ci-robot
```

Comment:
@dobbymoodge @stevekuznetsov this account needs to be created. I am not sure what's the email of the Github bot otherwise I would create it using that email.

Reply:
e-mail of this bot I can send you

Reply:
We could do basic auth in Apache for this account, either traditionally or by extracting credentials from the client certificate. We probably don't need to create a jenkins account for it.

Reply:
Is Apache doing auth for Jenkins via a module? 

Also, another question I forgot to ask today: do you have the current apache config somewhere in git (ci-config?)?

Reply:
+1 to `ci-config` for this